tests/ssl-opt.sh

```diff
@@ -3974,6 +3986,7 @@ run_test    "Authentication: server max_int chain, client default" \
             -C "X509 - A fatal error occurred"
 
 requires_full_size_output_buffer
+requires_config_value_at_least "MBEDTLS_X509_MAX_INTERMEDIATE_CA" 8
```

Comment:
Sorry @artokin, it should be `requires_config_value_exactly` here and below, too.

Reply:
no problem, updated in the next commit